src/main.tsx

```diff
@@ -225,25 +145,24 @@ Devvit.addTrigger({
     await redis.zRemRangeByScore(key, 0, expiredBefore);
     const count = await redis.zCard(key);
 
-    if(count < commentLimit) {
+    if (count < contentLimit) {
       await redis.zAdd(key, {
         member: contentId,
         score: new Date().getTime(),
       });
-
       return;
     }
 
-    const response = appSettings["comment-reply"] as string;
-    if(!!response) {
+    const response = (appSettings[`${type}-reply`] as string).trim();
+    if (!!response) {
       try {
         const comment = await reddit.submitComment({
           id: contentId,
           text: response,
         });
-        comment.distinguish();
-      } catch(e) {
-        if(e instanceof Error) {
+        await comment.distinguish();
```

Comment:
🔕 I didn't `await` this call on purpose because the rest of the trigger doesn't care about it

Plus, if distinguishing fails for some reason, we should still try to remove the comment

Reply:
In case it fails, though now that I think about it you can't really do anything in that case. Would there be any issues with the app instance ending before the comment was distinguished?

Reply:
Nah, auth matters when the request gets to the handler and there's a 10s grace-period after the request ends where the auth is still valid, so even in minor race-cases like this the risk of failure is basically zero

Reply:
Okay, I've undone that change.